coalib/bearlib/abstractions/Linter.py

```diff
@@ -165,6 +166,10 @@ def __repr__(cls):
                 cls.__name__, options['executable'])
 
     class LinterBase(metaclass=LinterMeta):
+        """
+        The ``LinterBase`` class is also a subclass of the ``LinterParent``
+        class.
+        """
```

Comment:
Document that inside the `@linter` documentation, not here :) `LinterBase` is not exposed outside.